pkg/controller/certificaterequests/venafi/venafi.go

```diff
@@ -88,7 +90,20 @@ func (v *Venafi) Sign(ctx context.Context, cr *cmapi.CertificateRequest, issuerO
 
 	duration := apiutil.DefaultCertDuration(cr.Spec.Duration)
 
-	certPem, err := client.Sign(cr.Spec.CSRPEM, duration)
+	customFields := []certificate.CustomField{}
+	if annotation, exists := cr.GetAnnotations()["venafi.cert-manager.io/custom-fields"]; exists && annotation != "" {
```

Comment:
Could you define a const for this annotation key within the v1alpha2 and v1alpha3 types.go file: https://github.com/jetstack/cert-manager/blob/master/pkg/apis/certmanager/v1alpha2/types.go && https://github.com/jetstack/cert-manager/blob/master/pkg/apis/certmanager/v1alpha3/types.go

This forms part of our API 😄